uws@0.13.0 is non-existent

It looks like uws@0.13.0 is not released on npm (see error below), could you please update to 0.14.1?  Thank you.

```
npm ERR! notarget No compatible version found: uws@0.13.0
npm ERR! notarget Valid install targets:
npm ERR! notarget 0.14.1
npm ERR! notarget
```

--- a/package.json
+++ b/package.json
@@ -18,7 +18,7 @@
     "sc-formatter": "~3.0.0",
     "sc-simple-broker": "~2.0.0",
     "uuid": "3.0.1",
-    "uws": "0.13.0",
+    "uws": "0.14.1",
     "ws": "1.1.1"
   },
   "devDependencies": {

